src/PaymentGateService/PaymentGateService.cpp

```diff
@@ -28,7 +28,7 @@
 #include "Logging/LoggerRef.h"
 #include "PaymentGate/PaymentServiceJsonRpcServer.h"
 
-#include "CryptoNoteCheckpoints.h"
+#include "../config/CryptoNoteCheckpoints.h"
```

Comment:
Is it possible to avoid the relative paths in the includes?  If someone moves a file, these will lead to a different location

Reply:
@who-biz we already discussed this with @nnian under another comment in this PR. All relative paths will be changed to something like `#include <module/header.h>` in one of the next PRs.

Reply:
Has anyone else noticed an error? Otherwise we would merge it for this time and change the relative paths later in a separate PR

Reply:
I have no other comments besides relative paths (that can be fixed in a separate PR). However I think we should also wait for PR #92 to complete, since I'll fix Travis CI issues within the scope of that PR.

Reply:
Oof sorry, I must have missed that somehow.